assets/js/googlesitekit/api/cache.js

```diff
@@ -0,0 +1,284 @@
+/**
+ * External dependencies
+ */
+import { isEqual } from 'lodash';
+
+let storageBackend;
+/**
+ * Override the storage backend.
+ *
+ * Largely used for tests. Should not be used directly.
+ *
+ * @param {*} backend Backend to set for the cache.
+ */
+export const _setSelectedStorageBackend = ( backend ) => {
+	storageBackend = backend;
+};
+
+let storageKeyPrefix = 'googlesitekit_';
+/**
+ * Override the key prefix used in storage.
+ *
+ * Largely used for tests.
+ *
+ * @param {string} keyPrefix String to prefix storage keys with.
+ */
+export const _setStorageKeyPrefix = ( keyPrefix ) => {
+	storageKeyPrefix = keyPrefix;
+};
+
+const defaultOrder = [ 'localStorage', 'sessionStorage' ];
+let storageOrder = [ ...defaultOrder	];
+/**
+ * Override the priority of storage mechanisms.
+ *
+ * Largely used for tests. Implicitly resets the selected storage backend,
+ * causing `_getStorage` to re-run its checks for the best available
+ * storage backend.
+ *
+ * @param {Array} order Ordered array of storage backends to use.
+ */
+export const _setStorageOrder = ( order ) => {
```

Comment:
Same here, let's remove this function. We shouldn't be introducing functions that are not part of the public interface and not used at all. If there really is a need for something like this, it should be introduced together with the code that indicates that need.